docs/develop/api-reference/index.mdx

```diff
@@ -485,6 +485,19 @@ The resulting incremented value must be between -9223372036854775808 and 9223372
 
 :::
 
+### Ping
+Sends a ping to the server. There are no parameters to the request.
```

Comment:
Context for why this command is useful would be nice here. What is an easy use case, checking for connectivity or determining health?

I'd also drop `There are no parameters to the request`